Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rate_extractor"
+version = "0.1.0"
+authors = ["Harish Mudgal <harish.m@surya-soft.com>"]
+edition = "2018"
+
+[lib]
+name="lib"
+path="src/lib.rs"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+calamine = "0.16.1"
+clap = "~2.27.0"
```

Comment:
@harishym can u tell me what is this ~ indicate?

Reply:
these are the crates available online at crates.io. we are importing these crates into my package so that i could make use of them.
~ indicates can be updated to higher version depending on what all field of the version we have provided. Here we have provided major, minor and patch. So only patch field can be updated.